packages/cdk8s-cli/lib/import/base.ts

```diff
@@ -23,16 +24,15 @@ export abstract class ImportBase {
   public abstract get moduleNames(): string[];
   protected abstract async generateTypeScript(code: CodeMaker, moduleName?: string): Promise<void>;
 
-  public async import(options: ImportOptions) {
+  public async import(options: ImportOptions, importSpec?: ImportSpec) {
```

Comment:
Why not put `moduleNamePrefix` in `ImportOption`?

Reply:
Ahh, I see it now. Missed it during the merge.

Reply:
And now the comment from above makes sense :)